src/screens/ChatScreen/AnonymousChatScreen.tsx

```diff
@@ -61,7 +61,7 @@ const AnonymousChatScreen = () => {
     selfAnonUserInfo,
     goBackFromChatScreen,
     goToChatInfoScreen,
-    sendChat,
+    sendChatMutation,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `sendChat` to `sendChatMutation` is a good move towards using react-query for mutations. This will provide benefits such as automatic retries, caching, and optimistic updates out of the box.

```diff
-    sendChat,
+    sendChatMutation,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->